src/lib.mli

```diff
@@ -241,12 +241,13 @@ module DB : sig
   val find : t -> Lib_name.t -> (lib, Error.Library_not_available.Reason.t) result
   val find_many
     :  t
+    -> loc:Loc.t
     -> Lib_name.t list
     -> lib list Or_exn.t
 
   val find_even_when_hidden : t -> Lib_name.t -> lib option
 
-  val available : t -> Lib_name.t -> bool
+  val available : t -> loc:Loc.t -> Lib_name.t -> bool
```

Comment:
Can `available` raise? If not, it doesn't seem worth passing it a location

Reply:
You're right, I removed it.